index.md

```diff
@@ -0,0 +1,80 @@
+# Vanilla Javascript in the Browser Workshop 
+ 
+Welcome to Vanilla Javascript in the Browser workshop hosted by [San Diego JS][san diego js].
+
+## Pre-event Setup Instructions
+
+Prior to your arrival the following should be installed on your system:
+
+0. [Install Git][git-scm]
+0. [Install Node.js 4.2 LTS][node-install]
+0. Setup NPM for non-sudo installation
```

Comment:
We'll also need to have Mocha and Chai(?) as well as Gulp installed globally, right?